### DISCUSSION TITLE
Way to wrap text automatically when using text?

### DISCUSSION THREAD

**Discussion by joaonunatings** [mProjectsCode/obsidian-meta-bind-plugin#77] [Q&A]
I know there is `class(meta-bind-full-width)` and `class(meta-bind-high)` but is there a way which it wraps to the text inside automatically without using `text_area`?

I tried creating a new class called `meta-bind-wrap` with `inline-size: min-content` as explained [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Text/Wrapping_Text) but did not work as expected.

**Comment by mProjectsCode** [comment]
I don't believe that is possible without `text_area`
